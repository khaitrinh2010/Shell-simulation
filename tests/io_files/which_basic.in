which var  
which ls
which notvalid notvalid2
