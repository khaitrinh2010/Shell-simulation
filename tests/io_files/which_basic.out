>> var: shell built-in command
>> /usr/sbin/ls
>> notvalid not found
notvalid2 not found
>> 
